src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const list = document.querySelector('.gallery__list');
+
+list.addEventListener('click', (events) => {
```

Comment:
```suggestion
list.addEventListener('click', (event) => {
```
You handle the single event, not plural. 
You can shorten it if there is a problem with it